prompt-service/src/unstract/prompt_service/plugins/table_extractor/src/processor/post_processor.py

```
@@ -0,0 +1,245 @@
+                "docker run --rm --memory=512m --cpus=1 --network=none "
+                f"--mount type=bind,source={script_path},target=/app/script.py,readonly "
+                f"--mount type=bind,source={tsv_path},target=/app/tsvdata.tsv,readonly "
+                "--workdir /app "
+                "gcr.io/distroless/python3-debian12:nonroot "
+                "script.py tsvdata.tsv"
+            )
+            logger.info(f"Running post processing in container: {cmd_str}")
+            cmd = cmd_str.split()
+
+        try:
+            result = subprocess.run(cmd, check=True, capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)